src/content/docs/apm/apm-ui-pages/errors-inbox/errors-inbox.mdx

```diff
@@ -68,6 +68,8 @@ Clicking on a specific error group takes you to the error details page, where yo
 
 The detailed view also allows you to view specific errors. In the top right, you can navigate between the first instance of the error, the last, and any instance inbetween.
 
+Click **Open in IDE** to use CodeStream to find the source of the error in your IDE.
+
```

Comment:
I would expand on this.  Maybe something like:
For errors with stack traces, click the **Open in IDE** button to use New Relic's integration with the [CodeStream](src/content/docs/codestream/start-here/codestream-new-relic.mdx) IDE extension to find the source of the error in your IDE and discuss resolution with your teammates.   The Open in IDE button requires that an application be associated with a Repository](Link to config) and telemetry to be associate to a commit SHA or release tag [Link to docs].